events/guildBanAdd.js

```diff
@@ -9,9 +9,9 @@ module.exports = {
             .setColor(DARK_RED)
             .setAuthor(`Membre banni`, ban.user.displayAvatarURL({dynamic: true, size: 4096, format: 'png'}))
             .setThumbnail(ban.user.displayAvatarURL({dynamic : true, size: 4096, format: 'png'}))
-            .setDescription(`<@${ban.user.id}>\n`)
+            .setDescription(`${ban.user}\n`)
```

Comment:
```suggestion
            .setDescription(`${ban.user}`)
```

Reply:
Passe plutôt par toString() du coup plutôt que utiliser des backticks, ça pèse moins lourd.

```suggestion
            .setDescription(ban.user.toString()`)
```